config/environments/production.rb

```diff
@@ -45,7 +45,6 @@
   config.cache_store = :redis_cache_store, {
     driver: :hiredis,
     url: ENV.fetch("OFN_REDIS_URL", "redis://localhost:6379/0"),
-    namespace: "cache",
```

Comment:
This wasn't needed to fix it, but I dropped it because it adds complexity with no value. Lots of our basic cache keys from things like `Spree::Config` will go through this, and it's a lot easier to debug those keys if they're not namespaced. E.g: the keys in the store match what you expect them to be, `"/spree/some_key"` on the app side will be `"cache:spree/some_key"` on the Redis side, which is pretty annoying if you're actually trying to manually check what's there.

When we add more uses to Redis (which I'm already planning) there are several other options for separating and namespacing different cache uses, and we'll need to use those anyway. For example, each Redis instances has 16 numbered "databases" by default, so you can use `redis://localhost:6379/0` to `redis://localhost:6379/16`. Secondly, if we start using it for Sidekiq, we'll actually need a separate Redis instance on a separate port, with slightly different configs in terms of persistance and eviction policies, which will mean an instance for jobs handling would be on `redis://localhost:6380/0` for example.

Reply:
I'm already envisioning 3 instances with 3 (necessarily) different sets of configs:
```
redis-cache: redis://localhost:6379/0
redis-jobs: redis://localhost:6380/0
redis-sessions: redis://localhost:6381/0
```

Reply:
poor single-VPS instances :see_no_evil: